projects/core/src/configurator/commons/facade/configurator-group-status.service.spec.ts

```diff
@@ -100,6 +101,20 @@ describe('ConfiguratorGroupStatusService', () => {
       expect(store.dispatch).toHaveBeenCalledWith(expectedAction);
     });
 
+    it('should get first uncompleted group', () => {
```

Comment:
Both **uncomplete** and **incomplete** are right, but we use in configurator.model.ts **incomplete** (see line 10)
Please, change accordingly not only the test description but also the name of the methode to getFirst**In**completeGroup